taskfiles/wasm.yml

```diff
@@ -21,4 +21,18 @@ tasks:
     status:
     - test -f "{{.JS_BUILD_ROOT}}/modules/rpc/service.js"
 
+  publish-local:
+    desc: Export the wasm-js public API
+    vars:
+      JS_ROOT: "{{.SRC_DIR}}/src/js"
+      JS_BUILD_ROOT: "{{.BUILD_ROOT}}/node/output"
+      NPM_CMD: "PATH={{.BUILD_ROOT}}/node/bin/:$PATH {{.BUILD_ROOT}}/node/bin/npm"
```

Comment:
since these are used in other tasks, they can be declared globally (move them to the top of the file)